lib/scala/pkg/src/main/scala/org/enso/pkg/QualifiedName.scala

```diff
@@ -31,7 +31,8 @@ case class QualifiedName(path: List[String], item: String) {
     * @return a [[QualifiedName]] with the updated project name
     */
   def renameProject(newName: String): QualifiedName = {
-    this.copy(path = newName :: path.tail)
+    val namespace = path.head
+    this.copy(path = namespace :: newName :: path.drop(2))
```

Comment:
I understand that we can assume that the `project` sugar is already not present here, correct?

Reply:
yes